lib/templates/src/components/Lobby.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div>
+    This is a lobby where participants wait for other participants to join so
+    that the number of required participants to start an interactive experiment
+    is met.
+  </div>
+</template>
+
+<script>
+import { joinLobby } from '@/socket.js';
```

Comment:
Perhaps it would be nice to expose all functionality relevant to the socket as part of `$exp`? That way implementers have a single mental concept of where they need to go to use a magpie functionality. I.e. `this.$exp.socket.joinLobby()`

Reply:
Sure, I did not try to put the socket module as part of the `magpie-base` library since the socket module also needs access to the store. If we solve the store access then this should definitely be possible.